### ISSUE TITLE
Transfer details shows "Source" tx hash AND "Destination" tx hash

### ISSUE THREAD

**Issue by tjhunt1** [icon-project/Nexus#685]
**Is your feature request related to a problem? Please describe.**
I think once a transfer has reached "Success" status, "Transfer Details" should show both the "Source chain" tx has AND the "Destination chain" tx hash.

Right now it only shows the tx hash from the "Source chain"

**Additional context**
<img width="1090" alt="Screen Shot 2022-08-10 at 4 05 19 PM" src="https://user-images.githubusercontent.com/98855020/184020127-cc8751ab-309e-4163-842d-776e8cd1521e.png">

**Comment by duyphanz** [comment]
cc @sanglt1902

**Comment by sanglt1902** [comment]
@tjhunt1 You mean, we should show the transaction hash once user perform transfer token and transaction hash confirm for this transfer. Am I correct?

**Comment by tjhunt1** [comment]
@sanglt1902 yes, preferably if it was a transfer from ICON -> BSC it would look like:

Transaction Hash (ICON)               0x47834....
Transaction Hash (BSC)                 0x5sj2d7....

**Comment by sanglt1902** [comment]
@tjhunt1 Let me explain a bit. Currently, we catch transactions and check whether these transactions were confirmed yet based on the events TransferStart (transaction creation) and TransferEnd (transaction confirmation). These events came from BTS and BMC smart contracts of the same chain, not from both chains.

Exp: User performs transfer 1 ICX from ICON -> BSC 
For this transaction, we catched two transactions hash from ICON
- One for Transaction Hash (ICON) 0x47834... (Transaction creation)
- One for Transaction Hash (ICON) 0x56278... (Transaction confirmation)
 
 
So that we cannot show the transactions as your expectation

> Transaction Hash (ICON) 0x47834....
> Transaction Hash (BSC) 0x5sj2d7....

Do you want to show two transactions hash as I explain above?

**Comment by tjhunt1** [comment]
Which is currently being displayed, transaction creation or transaction confirmation?

**Comment by sanglt1902** [comment]
> Which is currently being displayed, transaction creation or transaction confirmation?

it's transaction creation

**Comment by CyrusVorwald-ICON** [comment]
What are the events on the destination chain?

**Comment by sanglt1902** [comment]
> What are the events on the destination chain?

I'm not sure about this. Maybe we should have some advice from the SCORE team

**Comment by izyak** [comment]
When ICX is to be sent to BSC, 
txn is originated on ICON and emits Message event and Transfer Start event.
Relayer picks up the Message event, which as information on which chain to forward the message to, and makes a transaction on BSC. This generated Transfer Recieved event.
Once the transaction is successful on BSC, it emits successful message, which is picked up by relayer and forwards it to icon again, to acknowledge txn as been successful. This generated TransferEnd event. 
So, TransferStart and TransferEnd will come from source chain, and TransferRecieved from destination chain.

**Comment by sanglt1902** [comment]
> When ICX is to be sent to BSC, txn is originated on ICON and emits Message event and `Transfer Start` event. Relayer picks up the Message event, which as information on which chain to forward the message to, and makes a transaction on BSC. This generates `Transfer Recieved` event. Once the transaction is successful on BSC, it emits successful message, which is picked up by relayer and forwards it to icon again, to acknowledge txn as been successful. This generated `TransferEnd` event. So, `TransferStart` and `TransferEnd` will come from source chain, and `TransferRecieved` from destination chain.

@CyrusVorwald-ICON @tjhunt1 do you want us to collect transaction hash of TransferRecieved?

**Comment by CyrusVorwald-ICON** [comment]
I would prefer to close this in favor of https://github.com/icon-project/Nexus/issues/707

**Comment by tjhunt1** [comment]
#707 looks good to me
